### DISCUSSION TITLE
How to get lower-probability entities from NER?

### DISCUSSION THREAD

**Discussion by t-kevinyang** [explosion/spaCy#8673] [Help: Coding & Implementations] [answered]
```
nlp = spacy.load('en_core_web_sm')
nlp(text).ents
```
gives some entities, but is there an easy way to ask spacy to keep giving me more objects it thinks might be entities but which were a bit lower probability? Essentially I'm asking to lower the "threshold" (if that's how it's implemented under the hood) and rerun the NER to get more candidate entities.

**Comment by polm** [comment, marked as answer]
No, there is no way to do that. The spaCy NER component doesn't really work like that. 

You might want to take a look at the recently released [SpanCategorizer](https://spacy.io/api/spancategorizer); one of the main use cases for it was getting NER with per-annotation probabilities.
